esprima.js

```diff
@@ -6173,7 +6186,11 @@ parseYieldExpression: true, parseAwaitExpression: true
 
     function parseClassImplements() {
         var id, implemented = [], marker, typeParameters;
-        expectContextualKeyword('implements');
+        if (strict) {
+            expectKeyword('implements');
```

Comment:
Implements is a reserved word...but only in strict mode. Basically these little sprinked updates are to make our parsing of 'interface' work in strict mode (which module-mode implies)